md/work-execution-audit.md

```diff
@@ -0,0 +1,84 @@
+# Work execution audit
+
+Sometimes it is useful to understand what is currently happening in the platform, in particular for work execution, see [Work service mechanism](execution-sequence-states-and-transactions.md).
+
+This can be achieved using a specific logger called _Work execution audit_
+
+## What can be tracked
+
+### Abnormal work execution
+
+The logger produces a _Warning_ each time a work takes too much time to be executed or it was _rescheduled_ too much times.
+
+Example of warning log produced:
+```
+2019-02-11 14:54:14,541 WARN  [BONITA_WORK_AUDIT.EXECUTION] (Bonita-Worker-1-06) Potential abnormal execution detected - cause TOO_MUCH_EXECUTIONS. org.bonitasoft.engine.work.WorkDescriptor@3727bde7[uuid=0ca437df-cd1e-46f1-875a-a199cabbdf50,type=EXECUTE_FLOWNODE,tenantId=1,parameters={processDefinitionId=7836119026252730773, processInstanceId=2013, readyHumanTask=false, flowNodeInstanceId=80075},retryCount=5,executionThreshold=2019-02-11T13:54:14.539Z,executionCount=11,registrationDate=2019-02-11T13:53:43.224Z,abnormalExecutionDetected=true]
+2019-02-11 14:54:14,846 WARN  [BONITA_WORK_AUDIT.EXECUTION] (Bonita-Worker-1-01) Potential abnormal execution detected - cause TOO_MUCH_EXECUTIONS. org.bonitasoft.engine.work.WorkDescriptor@4c18babb[uuid=208b32fe-c191-4c9b-9966-95cadc75e33a,type=EXECUTE_FLOWNODE,tenantId=1,parameters={processDefinitionId=7836119026252730773, processInstanceId=2015, readyHumanTask=false, flowNodeInstanceId=80087},retryCount=5,executionThreshold=2019-02-11T13:54:14.846Z,executionCount=9,registrationDate=2019-02-11T13:53:43.506ZabnormalExecutionDetected=true]
+2019-02-11 14:54:15,084 WARN  [BONITA_WORK_AUDIT.EXECUTION] (Bonita-Worker-1-10) Potential abnormal execution detected - cause TOO_MUCH_EXECUTIONS. org.bonitasoft.engine.work.WorkDescriptor@257ed754[uuid=eb1e719d-1baa-42dc-9ddc-bfdadd7e0ebe,type=EXECUTE_FLOWNODE,tenantId=1,parameters={processDefinitionId=7836119026252730773, processInstanceId=2017, readyHumanTask=false, flowNodeInstanceId=80103},retryCount=5,executionThreshold=2019-02-11T13:54:15.084Z,executionCount=10,registrationDate=2019-02-11T13:53:43.808Z,abnormalExecutionDetected=true
+```
+
+We can wee:
+
+* the type of issue: `TOO_MUCH_EXECUTIONS` OR `TOO_MUCH_TIME_ELAPSED_SINCE_REGISTRATION`
```

Comment:
~TOO_MUCH_EXECUTIONS~ => TOO_MANY_EXECUTIONS

Reply:
That's right, documentation can be updated now. The code is going to be changed in bonita-engine as well